index.html

```diff
@@ -0,0 +1,25 @@
+<html>
+<head>
+  <title>
+    Pats Salmon Goods
+  </title>
+</head>
+<body>
+  <h1> Pats Salmon Cookies </h1>
+  <img id='salmon'src='https://raw.githubusercontent.com/codefellows/seattle-201d21/master/class-06-dom-objects-domain_modeling/lab/assets/salmon.png' alt='tasty salmon'>
+  <div>
+    customer reviews: 'The food is great!!!'
+    'Salmon cookies? Who thought they could be so good!'
+    '10 out of 10 so gooood'
+  </div>
+  <br />
+  <div>
+    business locations: 1st and Pike, SeaTac, Capitol Hill, Seattle Center, and Alki. We are open from 6am to 8pm daily.
+  </div>
+  <link href='style.css' rel='stylesheet' type='text/css'>
+  <script src='app.js'></script>
```

Comment:
you should import your JavaScript at the end of the body tags and your CSS at the end of the head tags.